scripts/build/termux_step_massage.sh

```diff
@@ -97,11 +111,29 @@ termux_step_massage() {
 	local HARDLINKS
 	HARDLINKS="$(find . -type f -links +1)"
 	if [ -n "$HARDLINKS" ]; then
-		termux_error_exit "Package contains hard links: $HARDLINKS"
+		if [ "$TERMUX_PACKAGE_LIBRARY" = "bionic" ]; then
+			termux_error_exit "Package contains hard links: $HARDLINKS"
+		elif [ "$TERMUX_PACKAGE_LIBRARY" = "glibc" ]; then
+			local declare hard_list
+			for i in $HARDLINKS; do
+				hard_list[$(ls -i "$i" | awk '{printf $1}')]+="$i "
+			done
+			local root_file
+			for i in ${!hard_list[@]}; do
+				root_file=""
+				for j in ${hard_list[$i]}; do
+					if [ -z "$root_file" ]; then
+						root_file="$j"
+						continue
+					fi
+					ln -sf "${TERMUX_PREFIX_CLASSICAL}/${root_file:2}" "${j}"
```

Comment:
There's also no control over which file will be full file, and which will be symlink. Find doesn't sort output so symlink vs full file will vary between builds

Reply:
Hard links are sorted - https://github.com/termux/termux-packages/pull/16901/files#diff-16d956212adc88d9e2d11e83f8f76549851ba7df6c22552146fd376176cb969cR119